python/ray/util/sgd/torch/torch_trainer.py

```diff
@@ -670,6 +670,17 @@ def step(self):
 
         You may want to override this if using a custom LR scheduler.
         """
+        # Maintain Trainable backwards compatibility with pre v0.8.7.
+        if self._is_overriden("_train"):
```

Comment:
@richardliaw I added this to keep backwards compatibility, but maybe it makes sense to just raise an error here instead.

Reply:
yeah, raise instead

Reply:
done